### PR TITLE
Fix fisher_callhome_spanish update (compatibility to eg1)

### PR DESCRIPTION
**Pull request by ftshijt**
@siddalmia pointed out some incompatibility to the fisher_callhome. The PR aims to fix those towards egs1

### CODE REVIEW

**Comment by ftshijt** [comment]
@siddalmia Hi, could you double-check the fix as well?

File: egs2/fisher_callhome_spanish/st1/local/data.sh [hunk 1]
```diff
@@ -68,16 +68,20 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
...
+    cp -r data/fisher_train data/train
     cp -r data/fisher_dev data/dev
+    rm data/dev/text.*.en
     # Use 1 reference as dev set
-    ln -sf text.lc.en.0 data/dev/text.lc.en
-    ln -sf text.tc.en.0 data/dev/text.tc.en
-    ln -sf text.lc.rm.en.0 data/dev/text.lc.rm.en
+    cp data/fisher_dev/text.lc.en.0 data/dev/text.lc.en
+    cp data/fisher_dev/text.tc.en.0 data/dev/text.tc.en
+    cp data/fisher_dev/text.lc.rm.en.0 data/dev/text.lc.rm.en
```
Comment (siddalmia) [inline egs2/fisher_callhome_spanish/st1/local/data.sh:86]: Can you verify if this part of the code runs fine if you run the data prep from scratch?

**Review by siddalmia — COMMENTED** [review]
Just one comment, otherwise looks good to me.

File: egs2/fisher_callhome_spanish/st1/local/data.sh (hunk 1 above)
Comment (ftshijt) [reply egs2/fisher_callhome_spanish/st1/local/data.sh:86]: Yeah, I initiated a new environment for this round of experiments to fix the corresponding issue

**Comment by siddalmia** [comment]
Sounds good! Thanks for the fix!

**Pull request merged**